scripts/manage-project.ts

```diff
@@ -84,18 +87,52 @@ const isVersionExists = async (version: string) => {
   );
 };
 
+const createOpenAPIVersionToUpload = async () => {
+  console.log(
+      colors.green(
+          "CREATING OPEN API FILE TO UPLOAD... PLEASE WAIT..."
+      )
+  );
+
+  const openApiPath = path.join(__dirname, "../reference/OpenAPI.json");
+  const openAPIContent = JSON.parse(
+      (await fsPromises.readFile(openApiPath)).toString()
+  );
+
+  const pathToTmp = path.join(__dirname, "../tmp");
+  if (!fs.existsSync(pathToTmp)) {
+    fs.mkdirSync(pathToTmp);
+  }
+
+  const pathToTmpReferenceToUpload = path.join(__dirname, "../tmp/referenceToUpload");
+  if (!fs.existsSync(pathToTmpReferenceToUpload)) {
+    fs.mkdirSync(pathToTmpReferenceToUpload);
+  }
+
+  const newOpenApiFile = { ...openAPIContent };
+  newOpenApiFile.openapi = "3.1.0";
+
+  await fsPromises.writeFile(
```

Comment:
to Ci chyba nie zadziała jak nie będziesz miał wcześniej folderów tmp i referenceToUpload. Mylę się? Brakuje Ci instrukcji:
- sprawdz czy fioldery są
- utwóż je jak ich nie ma

Reply:
Sa wyżej polecenia mkdir.